pkgs/development/python-modules/python-jsonrpc-server/default.nix

```diff
@@ -34,5 +34,6 @@ buildPythonPackage rec {
     description = "A Python 2 and 3 asynchronous JSON RPC server";
     license = licenses.mit;
     maintainers = [ maintainers.mic92 ];
+    broken = true;
```

Comment:
This one is not broken afaik. It is needed for the python-language-server,
which is important.

```
Build Status for nixpkgs.python36Packages.python-jsonrpc-server.x86_64-linux on unstable
✔ python3.6-python-jsonrpc-server-0.0.2 from 2018-10-24 - https://hydra.nixos.org/build/83221618
```

Tested for all python versions!

Reply:
Ok. python38 is actually broken. Disabled this one in ee6f5a32bb090ddacee9f40056a5598b2f10480a
Also backported: 9094d3592533ec3708414193ae1ce63a246a648b